packages/excalidraw/components/App.tsx

```diff
@@ -5238,7 +5246,11 @@ class App extends React.Component<AppProps, AppState> {
           originSnapOffset: nextOriginOffset,
         };
       });
-    } else if (!this.state.newElement) {
+    } else if (
+      !this.state.newElement &&
+      !this.state.selectedElementsAreBeingDragged &&
+      !this.state.selectionElement
+    ) {
```

Comment:
we shouldn't reset `snapLines` when we're dragging (the "when selecting" is just a perf optim as right now it's a noop)

If not checked, the snap lines flicker as they're constantly being set and reset